### ISSUE TITLE
Wrong values in Const column

### ISSUE THREAD

**Issue by f1ac** [mattiaverga/OpenNGC#23]
There are 162 values in Const column which are not referring to any constellation. Values are 'Se1' and 'Se2'. A quick search showed that objects with these values can belong to different constellations. Also there are 10 empty values for NonEx objects.

**Comment by mattiaverga** [comment]
'Se1' = 'Serpens Caput'
'Se2' = 'Serpens Caudi'

These belongings were added by querying coordinates by Ephem python module and I used the abbreviation used there. Maybe I should drop a note in the readme?

The NonEx objects have no coordinates, so obviously constellation column is empty.

**Comment by f1ac** [comment]
Strange, for example IC1101 has value of Se1, but in both [wikipedia ](https://en.wikipedia.org/wiki/IC_1101) and [HEASARC](https://heasarc.gsfc.nasa.gov/db-perl/W3Browse/w3table.pl?tablehead=name%3Dngc2000&Action=More+Options) it shows Virgo. I've noticed several such inconsistencies, so I thought Se1 and Se2 are not correct.

Objects marked an non-existent still have coordinates where they were thought to be present initially, for example in HEASARC coordinates and constellations are present for every object. Even some NonEx objects in OpenNGC have these values filled in:
![image](https://user-images.githubusercontent.com/13987066/109537310-69615880-7ac7-11eb-850a-04bb7cca206c.png)
So it could be worth adding these for historical reference and for consistency.

**Comment by mattiaverga** [comment]
It seems that Ephem is not precise in returning the constellation. For example, looking IC1101 coordinates in a planetarium, it's indeed in the Virgo constellation (by less than 4 arcmin).
I'll report the problem upstream, or I'll have to find another tool to map object coordinates to a constellation.

About NonEx objects, some of them have coordinates set in NED (which is the primary source for this catalog), some others are really non existent at all.

**Comment by msdemlei** [comment]
Well, with a TAP client you can run queries as follows on the TAP server
http://dc.g-vo.org/tap:

```
select ngc.name, abbrev from
openngc.data as ngc
join cstl.geo
on 1=contains(point(raj2000, dej2000), p)
```

-- this gives constellation abbreviations against the "center" position.

You could get extra fancy for the objects that have shapes, doing
something like:

```
select ngc.name, ivo_string_agg(abbrev, ', ') as cons from
openngc.shapes as ngc
join cstl.geo
on 1=intersects(deepest_shape, moc(10, p))
group by ngc.name
```

-- this would give you the constellations overlapped.  It fails right
now because the MOC library we're linking against in the database can't
deal with non-convex polygons.

If you'd like to do this sort of thing let me know and I'll add
MOCs externally transformed to the cstl.geo table.

**Comment by mattiaverga** [comment]
The PyEphem bug will not be worked out soon, so I've started a revision of the constellation column based on the output of TAP server.
I've already found several discrepancies from TAP output, so I cannot take it as source of truth and I need to manually verify the data.

Basically, I've run the query provided by @msdemlei selecting `ngc.name, ngc.constellation, abbrev`, then I've taken out those lines where ngc.constellation != abbrev (301 entries). I'm now verifying those one by one.

It may take some time...
